charts/openyurt/values.yaml

```diff
@@ -1,6 +1,9 @@
 imagePullSecrets: []
 imagePullPolicy: IfNotPresent
 
+##  if unspecified, will use built-in variable `.Release.Namespace`.
+namespace: openyurt-system
```

Comment:
how about keep consistency with the old version, so rename `openyurt-system` to `kube-system`?